docs/competition/matches/match-scoring.md

```diff
@@ -18,36 +18,51 @@ Note that this role is _not_ about making decisions related to the interpretatio
 
 **Responsible for being in the arena, saving robots and recording the end-state of matches. They may not bring in any subordinate volunteers.**
 
-In this role you will be assigned an arena and work with a partner. You will be allowed to enter the arena during matches if it is deemed necessary, however it is very important that you do not obstruct any robot or its vision and leave the arena as soon as possible. Each of you will be each given a score sheet<sup>1</sup> on a clipboard which you must both individually fill in fully by the end of the match. Anything that happens after the match has ended must not be counted as a score and the arena must not be changed until both scorers have finished. Once a match has finished it is your responsibility to take your completed score sheet to the score entry desk which will be near the arenas and you must return promptly to the arena in case you need to be found for further scoring questions. Teams are not allowed to enter the arena to collect their robots until after scoring has finished.
+You MUST read [the rules](https://www.studentrobotics.org/docs/rules/) before starting this role, as you will be enforcing them. Especially make sure you understandthe scoring rules (i.e. what counts as 'in' a zone), and how the arena should be laid out.
 
-<sup>1</sup>: The score sheets currently label the pink corner as purple. This isn't a major issue because they both start with the letter P, but it's something to be aware of.
+In this role you will work in the arena with a partner. You will be the only ones allowed in the arena during matches, however it is very important that you do not obstruct any robot or its vision, (especially by casting shadows over markers).
 
-As well as scoring, your responsibilities include making sure robots start in the right corner and switching off robots that do not automatically turn off after a match ends or when a team requests that you do so. You must also make sure that any arena props (tokens, for example) are in the correct place at the start of each match as detailed below. You are not responsible, however, for making sure a match starts with the correct number of teams -- that is down to the shepherds.
+Your responsibilities are:
+- Recording the state of the arena at the exact end of the match (see 'Scoring').
+- Ensuring robots start in the correct corner, and are entirely within their starting zone.
+- Ensuring no team members touch a robot after it has first moved. (see 'Teams touching robots').
+- Switching off robots when a team requests that you do so, or if they do not automatically turn off after a match ends.
+- Resetting the arena by moving props (tokens, for example) back to the correct place before the start of the following match. There is no mechanism for delaying the matches, so please request extra help if you are struggling to make the deadlines.
```

Comment:
Perhaps frame this as "after you and your partner have both recorded the end-state of the match", rather than "before the next match". The latter is (hopefully) obvious, I would rather we were explicit about the former.

Reply:
Fixed in 77a18d4ef30cac635bf80030b24927267ec56e31

Reply:
Fixed in 77a18d4ef30cac635bf80030b24927267ec56e31